src/System.Net.HttpListener/tests/Resources/System.Net.HttpListener.Tests.rd.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Directives xmlns="http://schemas.microsoft.com/netfx/2013/01/metadata">
+  <Library Name="System.Net.Tests.HttpListenerRequestTests">
+    <Namespace Name="System.Text" Dynamic="Required All" />
```

Comment:
It might not be necessary to enable it for the whole namespace , but since it's test only change  probably won't hurt.Leaving it to you to decide if you want to enable per type instead.